gameClientToGameWindowTests/errorMessage.spec.ts

```diff
@@ -60,16 +66,14 @@ readGames().forEach((game: string) => {
 
             await spin(page, consoleMessages);
 
-            // Wait 61s so the reality check timer kicks in
-            await page.waitForTimeout(61000);
+            // Wait 121s so the reality check timer kicks in
+            await page.waitForTimeout(121000);
```

Comment:
same suggestion here, underscores for times

Reply:
sure, done that